src/components/Verse/PlayVerseAudioButton.tsx

```diff
@@ -1,20 +1,21 @@
 import React from 'react';
 import { useDispatch } from 'react-redux';
 import { playVerse } from 'src/redux/slices/AudioPlayer/state';
+import Verse from 'types/Verse';
 
 import PlayIcon from '../../../public/icons/play-circle-outline.svg';
 import Button from '../dls/Button/Button';
 
 interface Props {
-  verseKey: string;
+  verse: Verse;
 }
 const PlayVerseAudioButton = (props: Props) => {
```

Comment:
A few comments here. 

1) I missed this in the previous review. We follow the pattern of destructuring props inside of the component parameters. E.g. 

```
const PlayVerseAudioButton = ({verse}: PlayVerseAudioButtonProps) => {

}
```

2) Rename the Props interface to be more specific (See comment above) 

3) If this button only cares about timestamps, we should be passing all of the verse object. It'll make our data flow bloated. For e.g. If we want the button to play audio starting from a certain time. It should take a *single* prop called `playFrom` in seconds. And then dispatch that value to the store.